app/views/vatDetails/details.scala.html

```diff
@@ -27,30 +27,44 @@
     currentPage = messages("vatDetails.title")
   )
 
-  <div class="grid-row">
+  <div class="grid-row form-group">
```

Comment:
This shouldn't really be used here as there are no inputs: http://govuk-elements.herokuapp.com/form-elements/
Are there alternate classes that could add padding?